src/core/js/views/drawerView.js

```diff
@@ -86,7 +86,14 @@ define([
             this._isCustomViewVisible = true;
             Adapt.trigger('drawer:empty');
             this.showDrawer();
+
             this.$('.drawer-holder').html(view);
+
+            var model = this.collection.findWhere({
```

Comment:
Page Level Progress doesn't provide any title at present. I suggest opening a separate issue if that's what you're referring to, thanks.